plugins/builder/osbuild.py

```diff
@@ -246,6 +249,17 @@ def __init__(self, task_id, method, params, session, options):
             self.client.http.verify = val
             self.logger.debug("ssl verify: %s", val)
 
+    def upload_meta_data(self, data: Dict, name: str):
+        fd = io.StringIO()
+        json.dump(data, fd, indent=4, sort_keys=True)
```

Comment:
Heresy. This should really be `2` :stuck_out_tongue_closed_eyes:

Reply:
Haha! I think I agree. ;)